fastlane/Fastfile

```diff
@@ -17,4 +17,69 @@ platform :ios do
       test | xcpretty
       ")
   end
+
+  desc "Runs the test suite and calculates code coverage percentages for each
+  module within the SDK"
+  lane :coverage do
+    MINIMUM_COVERAGE = 100.0
+
+    # Run the test lane to get coverage profdata files
+    unit_tests
+
+    # Ensure the code coverage directory is empty before creating new coverage
+    sh("rm -rf coverage")
+
+    # Use xcov to get the coverage percentages
+    xcov(
+      scheme: "PaymentsCore",
+      workspace: "PayPal.xcworkspace",
+      output_directory: "fastlane/coverage",
```

Comment:
This is more just a general question than anything, but with this running on CI, do we need an output? Or is this how the coverage report is generated/interpreted by CI?

Reply:
Good spot - we probably don't need that

Reply:
The coverage output is used on line 43 of this file to get the report JSON and iterate through the coverage data. The reason I went this route is because the default terminal output is minimal (it lists the top level coverage data for each framework, but nothing more granular). It has more detailed information in the reported html / json, so figured it'd be helpful to iterate through the JSON and print out the files / functions that are poorly tested

Reply:
Gotcha, that makes sense - thanks for the explanation @jonathajones!